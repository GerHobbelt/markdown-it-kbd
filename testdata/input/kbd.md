# Test

This combination is cool: [[alt]]+[[f4]].
