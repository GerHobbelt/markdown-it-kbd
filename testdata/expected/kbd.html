<h1>Test</h1>

<p>
	This combination is cool: <kbd>alt</kbd>+<kbd>f4</kbd>.
</p>
